chia/wallet/wallet.py

```diff
@@ -188,8 +190,10 @@ def make_solution(
         primaries: Optional[List[Dict[str, Any]]] = None,
         min_time=0,
         me=None,
-        announcements=None,
-        announcements_to_consume=None,
+        coin_announcements=None,
+        coin_announcements_to_consume=None,
```

Comment:
Maybe `_to_assert` since we haven't used `consume` to describe these for quite a while.